versioned_docs/version-3.0.0-beta/billing/wallet.md

```diff
@@ -5,63 +5,24 @@ sidebar_label: Wallet
 slug: /billing/wallet
 ---
 
-After creating an organization, the user will be given a starter pack by Spheron. The user must attach his/her wallet to reap the benefits of Spheron's Pro package.
+As per defualt you get a Strater pack from Spheron and you can upgrade the plan and attach your wallet for payment and billing. 
+
+The user must attach wallet benefits of Spheron's Pro package.
 
 ## Connect Wallet
 
 <font size="4"> <b>Step 1.</b> Go to Billing section and click on <code>Connect Wallet</code> button. </font> <br/><br/>
 
 <img src={require('@site/static/img/3.0.0/connect-wallet.png').default} />
 
-<font size="4"> <b>Step 2.</b> Choose your preferred token (<b>USDT</b>, <b>DAI</b>, <b>WMATIC</b>, <b>WETH</b>), network (<b>Polygon Mumbai</b>), and wallet (<b>Metamask</b>) on the pop-up modal.</font> <br/><br/>
-
-<img src={require('@site/static/img/3.0.0/connect-wallet-modal.png').default} />
-
-<font size="4"> <b>Step 3.</b> Metamask raises an alert message stating <code>I'm the owner of this wallet and want to attach it to the organization</code>. Sign the transaction. And your wallet is connected. </font> <br/><br/>
-
-<img src={require('@site/static/img/3.0.0/connect-wallet-metamask.png').default} />
-
-## Spheron Faucet
-
-A Faucet is a tool that provides a small number of funds to start using a cryptocurrency without having to buy some. Spheron has its own faucet app, which will send the selected **TEST** tokens straight into your wallet. The goal of this faucet is to make the onboarding to the Spheron Platform smoother.
-
-Spheron faucet sends **100 TEST USDT**, **100 TEST DAI**, **0.05 TEST WETH** or **70 TEST WMATIC** based on the chosen token. You can use it up to 3 times a day.
-
-Please visit Spheron [**Faucet**](https://faucet.spheron.network/) to get **TEST** tokens in your wallet, and get started with Spheron.
-
-## Allowance
-
-Spheron asks you to give a token allowance amount to ensure that your subscription is renewed every month. Unable to cut subscription may stop all the all services running in the platform. Please make sure, your wallet have enough balance.
-
-:::danger Overdue
+<font size="4"> <b>Step 2.</b> Choose your preferred Network and Token (<b>USDT</b>, <b>DAI</b>, <b>WMATIC</b>, <b>WETH</b>), network (<b>Polygon Mumbai</b>, <b>Arbitrum</b>, ), and wallet (<b>Metamask</b>) on the pop-up modal.</font> <br/><br/>
```

Comment:
We are not on polygon Mumbai anymore, we are now on polygon mainnet or just polygon will suffice

Reply:
Updated to just Polygon!